src/main/java/net/revelc/code/impsort/maven/plugin/AbstractImpSortMojo.java

```diff
@@ -365,8 +379,8 @@ protected void fail(String message, Throwable cause) throws MojoFailureException
         : new MojoFailureException(message, cause);
   }
 
-  static LanguageLevel getLanguageLevel(String compliance) {
-    if (compliance == null || compliance.trim().isEmpty()) {
+  static LanguageLevel getLanguageLevel(String compliance, boolean ignoreIrrelevantParseErrors) {
+    if (compliance == null || compliance.trim().isEmpty() || ignoreIrrelevantParseErrors) {
```

Comment:
I'm not sure why the presence of this new option means that the parser should ignore the user-specified compliance level.

Reply:
The motivation is to just have `<ignoreParseErrorsBelowImports>true</ignoreParseErrorsBelowImports>` in the `pom.xml` and never look back. That means that Java versions that would otherwise make the plugin fail because the version is not yet known to the parser, must be dealt with (fall back to LanguageLevel.POPULAR).

Reply:
I think it's reasonable to fall back to the POPULAR one when one is specified that is not recognized (which guarantees errors) AND you have this option set.

However, it should not fall back to POPULAR and override the user configuration just because you have this option set, when there's a recognized version.

Reply:
This is exactly how it is implemented after my rework:
```java
private static LanguageLevel parseLanguageLevel(String langLevel,
      boolean ignoreParseErrorsBelowImports) {
    return Stream.of(LanguageLevel.values()).filter(ll -> ll.name().equals(langLevel)).findFirst()
        .or(() -> ignoreParseErrorsBelowImports ? Optional.of(LanguageLevel.POPULAR)
            : Optional.empty())
        .orElseThrow(() -> new IllegalArgumentException("No enum constant "
            + LanguageLevel.class.getName().replace('$', '.') + "." + langLevel));
}
```